src/app/renderer/components/common/FileTree/FileTreeStyles.scss

```diff
@@ -43,6 +43,23 @@
           height: 10px;
         }
       }
+
+      &.variant {
+        line-height: 1.2rem;
+
+        a {
+          &.variantActive {
+            color: #333333 !important;
```

Comment:
Those colors should not be hard coded here. Pls add them to config.scss and use the variables here. E.g.
```
$color-text-secondary: #ccc;
$color-text-secondary-highlight: #333;
```

Reply:
done